src/actions/biblioActions.js

```diff
@@ -13,6 +13,16 @@ const restUrl = process.env.REACT_APP_RESTAPI;
 //const ateamApiBaseUrl = 'https://beta-curation.alliancegenome.org/';
 const ateamApiBaseUrl = process.env.REACT_APP_ATEAM_API_BASE_URL;
 
+const taxon_to_mod = {"NCBITaxon:559292": "SGD",
```

Comment:
There's a components/biblio/topic_entity_tag/TaxonUtils.js  This might be better there and imported ?  I'd welcome @valearna and @sweng66 feedback on this.   Possibly a function that works on getModToTaxon to reverse the mappings and generates this would be better.

Reply:
@ianlongden Agree with @azurebrd. We should get this mapping from the database by calling endpoint `${process.env.REACT_APP_RESTAPI}/mod/taxons/all` or calling functon "getModToTaxon" in src/components/biblio/topic_entity_tag/TaxonUtils.js / reverse the mapping / adding Homo sapiens to the mapping after.

Reply:
@ianlongden Agree with @azurebrd. We should get this mapping from the database by calling endpoint ${process.env.REACT_APP_RESTAPI}/mod/taxons/all or calling functon "getModToTaxon" in src/components/biblio/topic_entity_tag/TaxonUtils.js / reverse the mapping / adding Homo sapiens to the mapping after.

Reply:
using `getModToTaxon` sounds good.